ballerina/Package.md

```diff
@@ -1,14 +1,466 @@
 ## Package overview
 
-The `mssql` library is one of the external library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
+This module provides the functionality required to access and manipulate data stored in an MSSQL database.
```

Comment:
```suggestion
This package provides the functionality required to access and manipulate data stored in an MSSQL database.
```

Reply:
Fixed in [b02d390](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/163/commits/b02d39058c4c5fdd3204febac47ec271b869d81b)